IntegrationTests/ReactContentSizeUpdateTest.js

```diff
@@ -25,64 +24,67 @@ const reactViewHeight = 102;
 const newReactViewWidth = 201;
 const newReactViewHeight = 202;
 
-const ReactContentSizeUpdateTest = createReactClass({
-  displayName: 'ReactContentSizeUpdateTest',
-  _timeoutID: (null: ?TimeoutID),
-  _subscription: (null: ?EmitterSubscription),
+type State = {|
+  height: number,
+  width: number,
+|};
 
-  UNSAFE_componentWillMount: function() {
+class ReactContentSizeUpdateTest extends React.Component<{}, State> {
+  _timeoutID: ?TimeoutID = null;
+  _subscription: ?EmitterSubscription = null;
+
+  state = {
+    height: reactViewHeight,
+    width: reactViewWidth,
+  };
+
+  UNSAFE_componentWillMount() {
     this._subscription = RCTNativeAppEventEmitter.addListener(
       'rootViewDidChangeIntrinsicSize',
       this.rootViewDidChangeIntrinsicSize,
     );
-  },
-
-  getInitialState: function() {
-    return {
-      height: reactViewHeight,
-      width: reactViewWidth,
-    };
-  },
-
-  updateViewSize: function() {
-    this.setState({
-      height: newReactViewHeight,
-      width: newReactViewWidth,
-    });
-  },
+  }
 
-  componentDidMount: function() {
+  componentDidMount() {
     this._timeoutID = setTimeout(() => {
       this.updateViewSize();
     }, 1000);
-  },
+  }
 
-  componentWillUnmount: function() {
+  componentWillUnmount() {
     if (this._timeoutID != null) {
       clearTimeout(this._timeoutID);
     }
 
     if (this._subscription != null) {
       this._subscription.remove();
     }
-  },
+  }
 
-  rootViewDidChangeIntrinsicSize: function(intrinsicSize) {
+  updateViewSize = () => {
```

Comment:
The only call-site for this method is in `componentDidMount`:

```JavaScript
componentDidMount() {
    this._timeoutID = setTimeout(() => {
      this.updateViewSize();
    }, 1000);
  },
```

Given the way it's called, we don't need to bind this function.